development/tools/jobs/tester/jobsuite/suite.go

```diff
@@ -19,33 +19,40 @@ type JobConfigPathProvider interface {
 }
 
 // CheckFilesAreTested function
-func CheckFilesAreTested(repos map[string]struct{}, testedConfigurations map[string]struct{}, jobBasePath, subfolder string) func(t *testing.T) {
+func CheckFilesAreTested(repos map[string]struct{}, testedConfigurations map[string]struct{}, jobBasePath string, subfolders ...string) func(t *testing.T) {
```

Comment:
this change will make `subfolders` a valid empty slice if we don't pass the strings. Shouldn't it be defined as `subfolders []string` to make it required? I understand it would need a little more edits but I think this approach will be safer for future usages of this function.